src/main/webapp/authorize.html

```diff
@@ -9,11 +9,12 @@
     <meta charset="UTF-8">
     <title>Please Authorize!</title>
     <link rel="stylesheet" href="style.css">
-    <script src="dashboard-manager.js"></script>
   </head>
-  <body onload="handleLogin()">
+  <body>
     <button id="signinButton">You must first authorize or reauthorize!!</button>
-    <script>
+    <script type="module">
+      import {handleLogin, authorizeCallback} from './auth-flow.js';      
+      var auth2;
```

Comment:
It's recommended to use ```let``` for the variables need to be reassigned instead of ```var```.

https://google.github.io/styleguide/jsguide.html#features-use-const-and-let